spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/actions/TestNewRewriteDataFilesAction.java

```diff
@@ -177,6 +187,59 @@ public void testBinPackWithFilter() {
     assertEquals("Rows must match", expectedRecords, actualRecords);
   }
 
+  @Test
+  public void testBinPackWithDeletes() throws Exception {
+    Table table = createTablePartitioned(4, 2);
```

Comment:
Nit: Would it be worth while to add inline comments to explain these parameters (like `createTablePartitioned(4 /* ??? */, 2 /* ??? */)`? Right now, it's hard to tell immediately what these arguments are, but I'll leave the choice to you.

Reply:
I think this is not needed, because you can see the meaning of the parameters in Intellij